src/Test.UnitTests.Sarif/Core/LocationTests.cs

```diff
@@ -226,14 +226,14 @@ private void VerifyIdRoundTripFromObjectHelper(Location location, bool shouldSer
         {
             location.ShouldSerializeId().Should().Be(shouldSerialize);
             string jsonLocation = JsonConvert.SerializeObject(location);
-            jsonLocation.Contains(Id, StringComparison.InvariantCultureIgnoreCase).Should().Be(shouldSerialize);
+            jsonLocation.Contains(Id).Should().Be(shouldSerialize);
```

Comment:
>Contains [](http://example.com/codeflow?start=25&length=8)

.NET 4.8 does not provide a Contains overload that accepts a string comparison arg. We don't need case-insensitivity here and elsewhere in any case.